airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_backfills.py

```diff
@@ -92,10 +95,16 @@ def to_iso(val):
 class TestBackfillEndpoint:
     @provide_session
     def _create_dag_models(self, *, count=1, dag_id_prefix="TEST_DAG", is_paused=False, session=None):
+        bundle_name = "dags-folder"
+        orm_dag_bundle = DagBundleModel(name=bundle_name)
```

Comment:
Can the class use the fixture instead?

Reply:
PR updated. All test functions now add dag bundle with fixture. For adding dag bundle in helper functions, `DagBundleModel` is still used to avoid creating dag bundle unnecessarily.